specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/examples/AssessmentsMetadata/CreateAssessmentsMetadata_subscription_example.json

```diff
@@ -9,7 +9,7 @@
         "policyDefinitionId": "/providers/Microsoft.Authorization/policyDefinitions/26a828e1-e88f-464e-bbb3-c134a282b9de",
```

Comment:
Regard to `model validation` errors,  this property is readonly and must be removed from request.